omc3/correction/response_madx.py

```diff
@@ -83,31 +96,47 @@ def _generate_madx_jobs(
     delta_k: float,
     num_proc: int,
     temp_dir: Path
-) -> Dict[str, float]:
+) -> dict[str, float]:
     """ Generates madx job-files """
-    LOG.debug("Generating MADX jobfiles.")
+    LOG.debug("Generating MAD-X jobfiles.")
     incr_dict = {'0': 0.0}
-    vars_per_proc = int(np.ceil(len(variables) / num_proc))
+    compute_deltap: bool = ORBIT_DPP in variables
+    no_dpp_vars = [var for var in variables if var != ORBIT_DPP]
+    vars_per_proc = int(np.ceil(len(no_dpp_vars) / num_proc))
 
     madx_job = _get_madx_job(accel_inst)
+    deltap_twiss = ""
+    # This is here only for multiple iteration of the global correction
+    # By including dpp here, it means that if deltap is in variables and dpp is not 0, the orbit and tune magnets change
+    # We have to be very careful that DELTAP_NAME is not used ANYWHERE else in MAD-X
+    if compute_deltap:
```

Comment:
Comment into if statement or at least a blank line above, so it is clear that it doesn't refer to any of the code above